src/components/Dropdown/V2/Dropdown.Item.tsx

```diff
@@ -82,8 +83,13 @@ export class Item extends React.PureComponent<Props> {
 
   handleOnClick = (event: Event) => {
     const { onClick } = this.props
+    const state: any = this.props.getState()
 
-    onClick(event, { hasSubMenu: this.hasSubMenu() })
+    if (state && state.allowMultipleSelection && state.selectionClearer) {
+      onClick(state, event)
```

Comment:
@tinkertrain 🤔 Hmm!! Is there a reason the argument order is reversed here? `(state, event)` vs `(event, state)`

Reply:
oh, mmm, in dropdown.actions, I created `clearSelection` following `selectItem`, which is defined as: `export const selectItem = (state, event, eventTarget) => { //... }`

So it's going to be switched in one place or the other! 😁